### ISSUE TITLE
시스템20 성능 실험

### ISSUE THREAD

**Issue by februa22** [nlp-research/biaffineparser#63]
biaffine attention 수정

**Comment by februa22** [comment]
성능:

> 53843/53842 [==============================] - 678s 13ms/step - epoch: 6.0000 - loss: 0.2847 - uas: 0.9255 - las: 0.9076 - eval_loss: 0.3723 - eval_uas: 0.9155 - eval_las: 0.8926

**Comment by februa22** [comment]
```json
{
    "num_lstm_units": 400,
    "num_lstm_layers": 3,
    "arc_mlp_units": 500,
    "label_mlp_units": 100,
    "word_embed_size": 200,
    "pos_embed_size": 100,
    "optimizer": "adam",
    "learning_rate": 0.001,
    "decay_factor": 0.9,
    "num_train_epochs": 50,
    "train_filename": "data/sejong.train.csv",
    "dev_filename": "data/sejong.test.csv",
    "out_dir": "parser/model_iss63",
    "word_vocab_name": "word.pkl",
    "pos_vocab_name": "pos.pkl",
    "rel_vocab_name": "rel.pkl",
    "head_vocab_name": "head.pkl",
    "word_embed_file": "embeddings/words.pos.original.vec",
    "pos_embed_file": "embeddings/words.tag.original.vec",
    "word_embed_matrix_file": "embeddings/word_embed_matrix.txt",
    "pos_embed_matrix_file": "embeddings/pos_embed_matrix.txt",
    "embed_dropout": 0.33,
    "lstm_dropout": 0.33,
    "mlp_dropout": 0.33,
    "batch_size": 128,
    "device": "gpu",
    "debug": false,
    "inference_input_file": "",
    "inference_output_file": "parser/model_iss63/sejong.test.inference.tsv"
}
```

**Comment by februa22** [comment]
시스템20-1 성능 실험.

```python
self.arc_logits = tf.add(
    tf.divide(arc_logits, 2),
    tf.divide(arc_logits2, 2),
    name='arc_logits')
```

성능:

> 53843/53842 [==============================] - 326s 6ms/step - epoch: 7.0000 - loss: 0.2485 - uas: 0.9341 - las: 0.9177 - eval_loss: 0.3864 - eval_uas: 0.9170 - eval_las: 0.8919
